src/views/AddItem.jsx

```diff
@@ -12,24 +12,56 @@ const dayConverter = (text) => {
 	}
 };
 
-export function AddItem({ listId }) {
+export function AddItem({ listId, data }) {
 	const [itemName, setItemName] = useState('');
 	const [frequency, setFrequency] = useState('soon');
-	const [message, setMessage] = useState('');
+	const [itemMessage, setItemMessage] = useState('');
+	const [errorMessage, setErrorMessage] = useState('');
+
+	const clearMessage = (messageToAdjust) => {
+		setTimeout(() => {
+			messageToAdjust('');
+		}, 3000);
```

Comment:
**no need to change** anytime you find yourself using "magic numbers/strings" move them to a constant so you understand better the intent of the code, for instance, I know from the function name `clearMessage` that this resets "a message", which is ok! But could be improved to something like

```jsx
setTimeout(() => {
  /** implementation */
}, MESSAGE_RESET_TIMEOUT)
```